cxx-qt-gen/src/parser/mod.rs

```diff
@@ -0,0 +1,166 @@
+// SPDX-FileCopyrightText: 2022 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+pub mod qobject;
+
+use qobject::{find_and_merge_cxx_qt_item, find_qobjects, ParserCxxQtData};
+use syn::{token::Brace, ItemMod, Result};
+
+/// A struct representing a module block with CXX-Qt relevant Items into ParserCxxQtData
```

Comment:
```suggestion
/// A struct representing a module block with CXX-Qt relevant syn::Items parsed into ParserCxxQtData, to be used later to generate Rust & C++ code
```

Reply:
A little more detail:
```suggestion
/// A struct representing a module block with CXX-Qt relevant syn::Items parsed into ParserCxxQtData, to be used later to generate Rust & C++ code. Items that are not handled specially are passed through for cxx to process.
```